tools/lint/rules.py

```diff
@@ -339,6 +339,11 @@ class TentativeDirectoryName(Rule):
     to_fix = "rename directory to be called 'tentative'"
 
 
+class InvalidMetaFile(Rule):
+    name = "INVALID-META-FILE"
+    description = "The META.yml file contains an invalid structure"
```

Comment:
This lint error will happen both when the META.yml file cannot be parsed as YAML, and when it parses but doesn't pass the validation rules, right?

This description sounds most like it's about validation, not invalid YAML. Separating the two cases might make it easier to write the description, but for a single message how about...:

```suggestion
    description = "The META.yml is not a YAML file with the expected structure"
```

Reply:
I see what you mean. Thanks!